File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int slen = s.length();
        if (slen != t.length()) return false;

        vector<int> count(26);

        for (int i=0; i<slen; i++) {
            count[s[i]-'a']++;
            count[t[i]-'a']--;
        }

        for (int i: count) {
            if (i != 0) return false;
        }

        return true;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        # count = Counter(nums)
        count = {}

        for num in nums:
            count[num] = count.get(num, 0) + 1
        
        sorted_count = sorted(count.items(), key=lambda x: x[1], reverse=True)

        return [sorted_count[i][0] for i in range(k)]